cfpb/consumerfinance.gov
# Archiving: basic functionality

Adds the ability to mark a page as archived and displays banner on archived pages.

## Additions

- `is_archived` and `archived_at` fields on CFGovPage
- Archived banner template & less
- Conditional to display archived banner in `banner_top` block on `base.html` template


## How to test this PR

1. Check out branch
2. Run migration and gulp styles
3. Open a page in admin, mark it as archived on Configuration tab,  and check that it displays archived banner


## Screenshots

### Archived admin interface

<img width="1070" alt="Screen Shot 2020-09-17 at 12 29 08 PM" src="https://user-images.githubusercontent.com/778171/93518463-ace15500-f8e1-11ea-93e2-eeeef70140ea.png">

### Archived banner

<img width="1264" alt="Screen Shot 2020-09-17 at 12 29 40 PM" src="https://user-images.githubusercontent.com/778171/93518315-7e637a00-f8e1-11ea-8d88-1af9c9a0df08.png">


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
